packages/node/src/configure/SubqueryProject.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { RegisteredTypes } from '@polkadot/types/types';
+import {
+  ProjectNetworkConfig,
+  ReaderFactory,
+  parseProjectManifest,
+  ReaderOptions,
+} from '@subql/common';
+import { SubqlDatasource } from '@subql/types';
+import { GraphQLSchema } from 'graphql';
+
+import {
+  loadProjectFromManifest0_0_1,
+  loadProjectFromManifest0_2_0,
+} from '../utils/project';
+
+export type SubqlProjectDs = SubqlDatasource & {
+  mapping: SubqlDatasource['mapping'] & { entryScript: string };
+};
+
+export class SubqueryProject {
```

Comment:
When a util function dependes on some type in a module you know that something is wrong.  The dependency order shouldn't be like this.

Reply:
I suggest you pull loadProjectFromManifest0_0_1 and loadProjectFromManifest0_2_0 into here.
and `loadProjectFromManifest` to be renamed to `SubqueryProject.create()`